resources/js/components/user.vue

```diff
@@ -0,0 +1,357 @@
+<template>	
+		<div class="w-100 mt-2 py-3">
+          <a href="#" @click="createuser" class="btn py-3 mb-5 mr-2 px-4 text-white fs1 font1 p-success btn-lg pull-right" style="border-radius: 0.6rem">Create user <span class="text-white fa fa-chevron-down"></span></a>
+          <a href="#" @click="uploadstudent" class="btn py-3 mb-5 mr-2 px-4 text-white fs1 font1 p-success btn-lg pull-right" style="border-radius: 0.6rem">Upload Student <span class="text-white fa fa-chevron-down"></span></a>
+          <br>
+     
+          <div class="notification-table ">
+				<table id="usertable" class="table table-hover" @click="edituser">
+					<thead>
+						<tr id="cheadV">							
+							<th width="30%">user name</th>
+				            <th width="20%">user abbr.</th>	            
+							<th width="20%">Faculty</th>
+				            <th width="15%">date created</th>			            
+				            <th width="15%">Action</th>
+						</tr>
+					</thead>
+					<tbody v-if="tableLoaded">
+				        <tr v-for="(user, index) in createduser" :key="index">	         
+				            <td width="30%" :title="user.title">{{user.name}}</td>
+				            <td width="20%">{{user.code}}</td>	           
+				            <td width="20%">{{user.faculty_code}}</td>	           
+				            <td width="15%">{{user.updated_at}}</td>	            			            	           
+				            <td width="15%" >
+				            	<span class="ml-2 fa fa-edit pl-3  fs01 cursor-1" @click="edituser(createduser[index])" style="border-left: 1px solid #ccc;"></span>
+				            	<span class="ml-2 fa fa-trash pl-3  fs01 cursor-1" @click="deleteuser(user.id)"></span>
+				            </td>
+				        </tr>
+			    	</tbody>
+		    	</table>            
+          </div>
+        </div>
+</template>
+
+<script>
+	export default{
+		data(){
+			return{
+				createduser:null,
+				tableLoaded:false,
+				facultiesHTML: null,
+				faculties: null,
+				departments:null,
+				watchfacultyHtml:{value:null},
+				uploadlist:[],
+				listTrHtml:""
+			}
+		},
+		methods: {
+		  ProcessExcel:function(data) {
```

Comment:
Are we processing student upload from the frontend? I thought we are sending the CSV to the backend do they heavy work? @saidabdul80 ?

Reply:
yes sir i am validating the data before sending to backend. however am not sending csv file but sending the data in json to the backend. with this it doesn't have to waste time to send request for invalid data

Reply:
I wouldnt want you to stress yourself on the frontend for this, scared it may have a performance issue but if you prefer it this way, we can see to it, thoughts @bynalab?

Reply:
We had a plan though, you're the PM, should I review my code to fit?

Reply:
although i do not know of any performance issue that might arises, but i have a reason to have done that which is speed and good user experience. decision is within you as the PM of the Project.